### PR TITLE
add help us improve field to all registrations

### PR DESCRIPTION
**Pull request by kdachev97**
Added a new tick box in Registration Forms for both EN and BG.

Before:
Screenshot 2024-03-13 at 10 51 05
Screenshot 2024-03-13 at 10 50 58

After:
Screenshot 2024-03-13 at 10 49 01
Screenshot 2024-03-13 at 10 49 57

### CODE REVIEW

File: public/locales/bg/validation.json [hunk 1]
```diff
@@ -22,5 +22,6 @@
   "legal-entity": "Юридическо лице",
   "unique-field-violation": "Полето `{1}` със тази стойност вече съществува в платформата.",
   "payment-reference": "Невалиден формат на кода за плащане",
-  "eik-invalid": "Невалидно ЕИК"
-}
+  "eik-invalid": "Невалидно ЕИК",
+  "help-us-improve": "Искате ли да ни помогнете да подобрим платформата? Искаме да чуем вашето мнение? Отделете ни 30 мин за подобрите платформа на подкрепи.бг"
```
Comment (sashko9807) [inline public/locales/bg/validation.json:26]: It was discussed with the design team, it is better to replace `30 мин` with `30 минути`. 
Same goes for the EN translation, `30 mins to `30 `minutes`

File: src/components/common/form/HelpUsImproveField.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+export default function HelpUsImproveField({ name }: HelpUsImproveFieldProps) {
+  const { t } = useTranslation()
+  return (
+    <CheckboxField
+      name={name}
+      label={
+        <Typography variant="body2">
+          {t('validation:help-us-improve')}{' '}
+          {/* <ExternalLink href={routes.helpUsImprove}>
```
Comment (sashko9807) [inline src/components/common/form/HelpUsImproveField.tsx:20]: Please remove the commented line.

**Review by sashko9807 — CHANGES_REQUESTED** [review]
Well done. There are minor last-minute  changes to the text, mentioned below, please look at them.

This commit should also be supplemented, by changes in the backend as well, in order to keep track of who actually has agreed to help, so if you can please do.

Needed changes to the backend are:
 [Adding helpUsImprove field](https://github.com/podkrepi-bg/api/blob/master/schema.prisma#L33), to the prisma model

- Adding helpUsImprove property, to the [registerDto](https://github.com/podkrepi-bg/api/blob/master/apps/api/src/auth/dto/register.dto.ts#L53) 

If you have any further questions, regarding the requested changes feel free to contact me anytime.

Thank you for your contribution!

File: public/locales/bg/validation.json (hunk 1 above)
Comment (ani-kalpachka) [reply public/locales/bg/validation.json:26]: I would update it to:
"Искате ли да ни помогнете да подобрим платформата? Искаме да чуем Вашето мнение? Отделете ни 30 минути, за подобрите платформа на Подкрепи.бг""

**Review by ani-kalpachka — APPROVED** [review]

**Review by sashko9807 — APPROVED** [review]

**Review by sashko9807 — CHANGES_REQUESTED** [review]
The automatic linter appears, to have failed. Do you mind running `yarn lint`(on the frontend repo), and commit the changes?

File: public/locales/bg/validation.json (hunk 1 above)
Comment (wqweto) [reply public/locales/bg/validation.json]: "Искаме да чуем вашето мнение" and "We want to hear from you" are plain statements and don't need question mark unless the original intend was to read "**Искате** да чуем вашето мнение?" and "**You** want us to hear from you?"

Comment (kdachev97) [reply public/locales/bg/validation.json]: @wqweto Този текст е вече променен в друг тикет: https://github.com/podkrepi-bg/frontend/pull/1751/commits/dbfc1483a55d467399012de75d344f08d3fe665b

**Pull request merged**